README.md

```diff
@@ -1,16 +1,24 @@
-# Title
+# IRB
 
 ## Learning Goals
 
--SWBAT 1
--SWBAT 2
++ Explain why REPLs are useful
```

Comment:
This might evolve as the video content develops, but it's a good rule of thumb to be detailed about why something like this is useful directly in the learning goals. There might be some more goals that can be pulled out of it.